### PR TITLE
Remove preview config

### PR DESCRIPTION
**Pull request by timja**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-916

### Change description ###
Tested in SAAT, also updated DB config for production

```
{
  "administratorLogin": "draftstore",
  "earliestRestoreDate": "2019-01-22T20:40:18.453000+00:00",
  "fullyQualifiedDomainName": "rpe-draft-store-saat.postgres.database.azure.com",
  "id": "***",
  "location": "uksouth",
  "name": "rpe-draft-store-saat",
  "resourceGroup": "rpe-draft-store-data-saat",
  "sku": {
    "capacity": 2,
    "family": "Gen5",
    "name": "GP_Gen5_2",
    "size": null,
    "tier": "GeneralPurpose"
  },
  "sslEnforcement": "Enabled",
  "storageProfile": {
    "backupRetentionDays": 35,
    "geoRedundantBackup": "Enabled",
    "storageMb": 51200
  },
  "tags": null,
  "type": "Microsoft.DBforPostgreSQL/servers",
  "userVisibleState": "Ready",
  "version": "10"
}
```

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -58,15 +33,15 @@ module "api" {
...
-    MAX_STALE_DAYS_CRON         = "${var.max_stale_days_cron}"
+    MAX_STALE_DAYS_DEFAULT = "${var.max_stale_days_default}"
+    MAX_STALE_DAYS_CRON    = "${var.max_stale_days_cron}"
 
-    FLYWAY_URL                  = "jdbc:postgresql://${var.db_host}:5432/draftstore${local.db_connection_options}"
-    FLYWAY_USER                 = "draftstore"
-    FLYWAY_PASSWORD             = "${data.azurerm_key_vault_secret.db_password.value}"
+    FLYWAY_URL      = "jdbc:postgresql://${var.db_host}:5432/draftstore${local.db_connection_options}"
+    FLYWAY_USER     = "draftstore"
+    FLYWAY_PASSWORD = "${data.azurerm_key_vault_secret.db_password.value}"
```
Comment (doncem) [inline infrastructure/main.tf]: oops, formatting gone. ah, not gone. there are single line separations around those blocks. as you were

File: infrastructure/main.tf [hunk 2]
```diff
@@ -76,29 +51,31 @@ module "api" {
...
-  sku_name            = "GP_Gen5_2"
-  sku_tier            = "GeneralPurpose"
-  common_tags         = "${var.common_tags}"
+  source                = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product               = "rpe-${var.product}"
+  location              = "${var.location_api}"
+  env                   = "${var.env}"
+  database_name         = "draftstore"
+  postgresql_user       = "draftstore"
+  postgresql_version    = "10"
```
Comment (doncem) [inline infrastructure/main.tf]: will module changes keep the current db in place?

File: infrastructure/main.tf (hunk 1 above)
Comment (timja) [reply infrastructure/main.tf]: i ran `terraform fmt` over the file, changes weren't much so didn't separate to a different PR

File: infrastructure/main.tf (hunk 2 above)
Comment (timja) [reply infrastructure/main.tf]: no I renamed the database so it will get deleted and recreated
this database isn't in use in production

**Pull request merged**